src/mpid/ch4/src/ch4i_comm.c

```diff
@@ -0,0 +1,1118 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *  (C) 2016 by Argonne National Laboratory.
```

Comment:
I don't know the policy, but should the newly-created files have the up-to-date year here (2018)?

Reply:
Good point, I will update it.